cmd/psa/verify_as_relyingparty.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package psa
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/veraison/apiclient/verification"
+	"github.com/veraison/evcli/common"
+	"github.com/veraison/psatoken"
+)
+
+var (
+	relyingPartyTokenFile *string
+	relyingPartyAPIURL    *string
+)
+
+var (
+	relyingPartyVeraisonClient common.IVeraisonClient = &verification.ChallengeResponseConfig{}
+	relyingPartyCmd                                   = NewRelyingPartyCmd(common.Fs, relyingPartyVeraisonClient)
+)
+
+func NewRelyingPartyCmd(fs afero.Fs, veraisonClient common.IVeraisonClient) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "relying-party",
+		Short: "Emulate a Relying Party",
+		Long: `
+	This command implements the "relying party mode" of interaction, where the
+	relying party was the challenger, and therefore the nonce is provided by the
```

Comment:
```suggestion
	relying party exchanges a well formed signed PSA attestation token with the Veraison verifier to perform attestation. Please note that the well formed PSA token is formed prior to this action possibly by a previous invocation of "evcli psa create".
```

Reply:
also done, though in a slightly different way